pkg/providers/snow/entry.go

```diff
@@ -0,0 +1,78 @@
+package snow
+
+import (
+	"context"
+
+	"github.com/aws/eks-anywhere/pkg/aws"
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/filewriter"
+	"github.com/aws/eks-anywhere/pkg/networkutils"
+	providerValidator "github.com/aws/eks-anywhere/pkg/providers/validator"
+)
+
+type ConfigManagerEntry struct {
```

Comment:
Nit:
```suggestion
type ConfigManager struct {
```

Since from the outside this will read as `snow.ConfigManager`